docs/interactions/structured-messages/list.md

```diff
@@ -43,10 +43,16 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 | **`structured_content.type`** | String | Type of the structured message. Must be "select". |
 | **`structured_content.title`** | String | Title of the list structured message. Limited to 1024 characters. |
 | **`structured_content.subtitle`** | String | **Optional**. Subtitle of the list structured message. Limited to 512 characters. |
-| **`structured_content.items`** | Array | An array of items representing the options presented to the customer. A maximum of 13 items is supported. |
+| **`structured_content.sections`** | Array | **Optional**. An array of sections in which the items will be organized.<br>Limited to 10 elements.<br>If blank, every item will be part of the same section. |
+| **Section Settings** | | |
+| **`structured_content.sections.title`** | String | **Optional if there's only a single section**. The title of the section.<br>Limited to 24 characters. |
+| **`structured_content.sections.identifier`** | String | Identifier of the section that will be used to organize items in the section.<br>Limited to 200 characters. |
+| **`structured_content.items`** | Array | An array of items representing the options presented to the customer. A maximum of 10 items is supported. |
 | **Item Settings** | | |
-| **`items.title`** | String | The title of the item. Limited to 80 characters. |
-| **`items.payload`** | String | **Optional**. Payload that will be returned with the structured response. Limited to 1000 characters. |
+| **`structured_content.items.title`** | String | The title of the item. Limited to 80 characters. |
+| **`structured_content.items.payload`** | String | **Optional**. Payload that will be returned with the structured response. Limited to 200 characters.<br>Automatically gets populated as a random hex if blank. |
+| **`structured_content.items.subtitle`** | String | **Optional**. The subtitle of the item. Limited to 100 characters. |
+| **`structured_content.items.section_identifier`** | String | **Optional if there's no section**. The identifier of the section where the item is.<br>If there's no section, the section_identifier field should be removed.<br>Each section must have at least 1 item.<br>Limited to 200 characters. |
 
```

Comment:
Move more attributes common to ABC and WA and supposed to be in all List sm in Primary attributes, fixed some validations numbers

![Screenshot from 2022-02-02 18-40-41](https://user-images.githubusercontent.com/1233351/152207936-530c5cdd-cf99-4bc4-8ef3-3b96ea8eb87e.png)